pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -796,9 +806,13 @@ func testEcdhAES() error {
 	return nil
 }
 
-func getDevicePrivateKey() (*ecdsa.PrivateKey, error) {
-	// XXX:TBD, currently only one private key
-	keyPEMBlock, err := ioutil.ReadFile(types.DeviceKeyName)
+// software private key
+func getEveNodePrivateKey(isDeviceKey bool) (*ecdsa.PrivateKey, error) {
```

Comment:
The old name is correct; the new name is very confusing.

Reply:
If you want a readPrivateKey(filename string) to avoid duplicating code that's fine. getDevicePrivateKey and getDeviceECDHPrivateKey can call that function.